### ISSUE TITLE
can not install on Win10

### ISSUE THREAD

**Issue by helixg** [DocNow/hydrator#38]
When I install the Hydrator-Setup-0.0.6.exe on Win10 Professional Edition,
There is a javascript error occur
like this

![hi1.png](https://i.loli.net/2020/05/03/tjAiZCK5Q7ylNmf.png)

**Comment by edsu** [comment]
Thanks for reporting this. I think this is your first time running hydrator? I neglected to test with a clean slate where Hydrator had not been run before.

I think I've fixed the problem. Can you give the just released v0.0.7 a try?

**Comment by edsu** [comment]
I didn't mean to close this issue.  Can you can confirm v0.0.7 doesn't have the same defect?

**Comment by helixg** [comment]
> @helixg I'm sorry for prematurely closing this issue. Can you can confirm that v0.0.7 fixes this problem?

It seems that the v0.0.7 can launch successfully

**Comment by helixg** [comment]
It seems that I need a pin to connect to my API key,  Can I enter the Access Key and Access Key Secret directly?

![hi2.png](https://i.loli.net/2020/05/04/sjXapw7V6tBS3yn.png)

**Comment by edsu** [comment]
When you click to link the account does your browser not open to a page at twitter asking for you approval to use your account ?On May 3, 2020 10:11 PM, helixg <notifications@github.com> wrote:
It seems that I need a pin to connect to my API key,  Can I enter the Access Key and Access Key Secret directly?

**Comment by hamburgersct** [comment]
> It seems that I need a pin to connect to my API key, Can I enter the Access Key and Access Key Secret directly?
> 
> ![hi2.png](https://camo.githubusercontent.com/676cae192c1622120ee43c3bd804a22c5ff91f6c/68747470733a2f2f692e6c6f6c692e6e65742f323032302f30352f30342f736a5861707737563674425333796e2e706e67)

Got this, too. Have you solved this?

**Comment by edsu** [comment]
@hamburgersct no, unfortunately not. it seems to only be an issue on some Win 10 installs. Something is preventing the browser window from opening. If you want we could try placing the url text for you to click on or copy/paste in the window. Would you be willing to try that?

**Comment by hamburgersct** [comment]
Yes, I'd like to try that. Thanks a lot for your reply!

**Comment by edsu** [comment]
Ok, give me a few hours to try something out and put a release together.

**Comment by edsu** [comment]
It took me a few more hours than I thought to get to it, but v0.0.8 is ready for you to try. It should display the Twitter URL you will need to point your browser at if Hydrator is unable to open the browser itself.

Can you give it a try?

https://github.com/DocNow/hydrator/releases/tag/v0.0.8

**Comment by hamburgersct** [comment]
![image](https://user-images.githubusercontent.com/32610103/82568972-2aa28780-9bb2-11ea-8cae-254dde25596d.png)
Thanks for your help!☺ However, it still didn't work.
I tried to click the button, but nothing happened. No new window in my browser and no URL was displayed.

**Comment by edsu** [comment]
DId you see the URL to use in your browser to get the PIN?

**Comment by edsu** [comment]
You say nothing happened but I see invalid PIN there in the screenshot. It would be super helpful if you could do a screencast (video) of what you are doing.

**Comment by hamburgersct** [comment]
[Sreenshot_video.zip](https://github.com/DocNow/hydrator/files/4663055/Sreenshot_video.zip)
Hi, it worked just like in this video. The "Invalid PIN" seemed to appear as long as I opened the application. Nothing happened when I click the Link button.

**Comment by edsu** [comment]
Thanks that is actually helpful, has that Invalid PIN message always appeared there? I just uploaded [v0.0.9](https://github.com/DocNow/hydrator/releases/tag/v0.0.9) which should make a Reset button visible. Can you click Reset, and try to link again?

https://github.com/DocNow/hydrator/releases/tag/v0.0.9

**Comment by hamburgersct** [comment]
[Screenshot_vid_2.zip](https://github.com/DocNow/hydrator/files/4663158/Screenshot_vid_2.zip)
Unfortunately, it still didn't work properly. No URL appeared there. But the Invalid PIN didn't appear anymore after I clicked Reset.
I will try twarc, so don't worry ;-)
Anyway, thank u Ed!!!

**Comment by edsu** [comment]
Argh!

One last thing we can try if you have some familiarity with the command prompt on Windows is to open a Windows command prompt, navigate to where your Hydrator is installed and run it manually:

```
cd AppData/Local/Programs/Hydrator
./Hydrator.exe
```

When you try to link the account does any kind of error message get displayed in the terminal? It's possible your Hydrator.exe is installed somewhere else. Right clicking on the Hydrator icon on your desktop and selecting Properties may help you locate it.

**Comment by hamburgersct** [comment]
Wow! You really helped me a lot! From the message in the command line, I found some errors with my network. As soon as I solve that, I believe that it can work properly. Really really thank you!😁

**Comment by edsu** [comment]
NICE! what was the network problem -- does it look like it is firewall related? Please let me know if you are able to find a fix!

**Comment by hamburgersct** [comment]
Hi! Recently I've tried to use twarc ( but I believe Hydrator is the same) and now I realize the problem:
**These two tools cannot work properly in China because of the GFW.** 
So, if someone in China needs to use these tools, he/she should try to overcome the network problem, or ask friends in other countries to help.
Anyway, thank you for developing these two wonderful tools!🤩

**Comment by edsu** [comment]
Oh wow, so you are behind The Great Firewall!? You will need to use a VPN right? If you get a chance I would love to see the error message you got in the terminal so that the Hydrator can provide a useful error message.

**Comment by 1109LLL** [comment]
<img width="567" alt="Screenshot 2020-12-08 at 22 22 05" src="https://user-images.githubusercontent.com/43610472/101495712-cf947e80-39a3-11eb-9308-140b24d4fca2.png">
Hi, after reading through this thread I think I have the same issue as @hamburgersct. Using a VPN doesn't help, it simply returns with a timeout error.

**Comment by edsu** [comment]
It sounds like your VPN isn't doing what you think then :-( One way of checking is to visit https://api.twitter.com/ in your browser to see if you get a page not found error from Twitter. If it just sits there waiting for a response you are blocked. I'm sorry but neither Hydrator or twarc punch through firewalls.

**Comment by 1109LLL** [comment]
![Screenshot 2020-12-08 at 22 34 07](https://user-images.githubusercontent.com/43610472/101497059-80e7e400-39a5-11eb-9ebf-3c6695109ec7.png)
agghhh, yep its blocked... Thanks for your help!

**Comment by edsu** [comment]
Actually that means you were able to talk to the server ok if you can see that error page.

**Comment by YuNFAnz06** [comment]
Have you solved this problem now? I really want to use this data set, but I don't know what to do.
